src/main/java/seedu/address/logic/commands/KeyCommand.java

```diff
@@ -0,0 +1,71 @@
+package seedu.address.logic.commands;
+
+//@@author lohtianwei
+import java.nio.charset.StandardCharsets;
+import java.util.logging.Logger;
+
+import com.google.common.hash.Hashing;
+
+import seedu.address.commons.core.LogsCenter;
+import seedu.address.logic.commands.exceptions.CommandException;
+import seedu.address.model.UserPrefs;
+
+/**
+ * Sets lock in model to true/false depending on current state
+ */
+public class KeyCommand extends Command {
+    public static final String COMMAND_WORD = "key";
+    public static final String COMMAND_ALIAS = "k";
+
+    public static final String MESSAGE_USAGE = COMMAND_WORD + ": Toggles the lock on MTM. "
+            + "Functionality of MTM reduced\n"
+            + "Input empty password to see current lock state\n"
+            + "Parameters: [PASSWORD]\n"
+            + "Example: " + COMMAND_WORD
+            + " myPassword";
+
+    public static final String MESSAGE_SUCCESS = "MTM lock toggled!";
```

Comment:
Do you think it's better to indicate the status from and to what instead of toggled? So that user don't have to enter `key`, to determine the toggled state?